### ISSUE TITLE
text on gas tanks is crowed when non-English language is selected

### ISSUE THREAD

**Issue by sanjuroj** [mercycorps/TolaActivity#1021]
Because many non-English languages take more space than English, the gas tank text on the program page becomes crowded and a bit confusing when e.g. Spanish is the selected language.

English looks nice:
![screen shot 2019-01-03 at 3 34 37 pm](https://user-images.githubusercontent.com/6685499/50667451-30241100-0f6e-11e9-8071-e8fff0a8239e.png)

Spanish not so nice:
![screen shot 2019-01-03 at 3 38 38 pm](https://user-images.githubusercontent.com/6685499/50667457-34502e80-0f6e-11e9-85a9-6b243265cca8.png)

**Comment by jennyhavah** [comment]
Yep, was afraid of this. @axoplasm in addition to playing with the horizontal spacing, I wonder if we could also reduce the line spacing such that there's noticeably less line spacing than there is "paragraph" spacing (i.e. between metric strings.)

I also wonder if it would be a good idea to force the ? never to wrap onto its own line by not having a space between the final letter and the icon -- or if that would just compound the issue.

**Comment by axoplasm** [comment]
IANAFSS (I Am Not A Fluent Spanish Speaker) but we could probably omit _están_. We can also reduce the font size a little but I'm not nuts about it. Those are band-aids, though, not long-term fixes.

A good long term fix might be to get as many app labels as possible to contain only the minimum possible text. I'm beginning to appreciate why signs in airports are all numbers & pictures 🛃

With pictograms this one might be:

![image](https://user-images.githubusercontent.com/169060/50783496-e97d3200-125f-11e9-8b03-1dc6fc007e1c.png)

…then the help text would explain what that value is. (Consider also that most folks will only need to have this graph explained a few times)

Re: wrapping the (?) — yes we should do that.

Re: line spacing: there's an implicit conversation here about vertical rhythm (...ie. difficulty of maintaining versus weird uncanny valley effect of not getting it 100% correct) but the short answer is, these aren't paragraphs, and in some cases they'll run into each other or even overlap. Reducing the font size and closing the line-height (as above) might help

**Comment by jennyhavah** [comment]
> IANAFSS (I Am Not A Fluent Spanish Speaker) but we could probably omit están. We can also reduce the font size a little but I'm not nuts about it. Those are band-aids, though, not long-term fixes.

I did experiment with not using are / están / sont but thought it read too much like an x > y mathematical statement. If you have something like:

**13% > 15% above target** it scans like "13% is greater than 15%" ... which isn't what we mean at all, plus it reads like a calculation error.

### I like the pictogram approach!
I have some questions / suggestions for refinement.

1. All the circle icons in close proximity are a bit too similar -- every icon is some variation of a circle, and then there are the question marks in circles. Can we visually differentiate the icons more? For example, instead of the tiny up and down arrows inside of circles, could we have larger arrows with no circle? Like: 
![image](https://user-images.githubusercontent.com/33670923/51130283-7f293c00-17e1-11e9-9929-fe6dcdbad3a9.png)

2. The target icon is a bit difficult to make out. Maybe see if a different option (like the two-circle variety) works better, if there's a little more contrast in between the concentric circles?

3. I was mulling over how we could integrate the explanation of the pictogram (all the phrases that used to be next to the graph) as well as the existing explanation of "on track", and also get rid of those question mark icons so there aren't as many competing icons. Here's what I came up with:

![image](https://user-images.githubusercontent.com/33670923/51131583-062be380-17e5-11e9-8922-4933bae292aa.png)

You click the pictogram to get the tool tip. You click "View indicators" to activate the filter. (Until we  do #948, the button would not appear on the home page version.)

**Comment by camgnostic** [comment]
Minor quibble - direction of change doesn't actually affect whether the indicator is on track, just which "not on track" bucket it's in if it's not... dunno if that matters but that leaps out at me from the italic explainer there

**Comment by camgnostic** [comment]
I mean, I don't think it's hugely important.  But maybe different text for the three elements (above, on track, under)?  And so then on-track doesn't need a DOC note, and above/under can have a parenthetical like "An indicator is over target if the result is more than 115% of the target (or less than 85% if the direction of change is negative"?  I worry I'm nitpicking here

**Comment by jennyhavah** [comment]
@camgnostic, I think that's a great idea! How about this text for the different variations:

# Unavailable
An indicator is unavailable if it has no targets, no completed target periods, or no results reported.

# Indicators not on track (above target version)
An indicator is not on track if the result is more than 115% of the target.

# Indicators on track
An indicator is on track if the result is no less than 85% of the target and no more than 115% of the target.

# Indicators not on track (below target version)
An indicator is not on track if the result is less than 85% of the target.

**Comment by jennyhavah** [comment]
@axoplasm Did I hear you say that you'd like to see our gauges converted to React prior to tackling this work? Should we add a separate ticket for that?

**Comment by axoplasm** [comment]
Is this in Satsuma? I can add this to the pile of design I'm doing there.

**Comment by jennyhavah** [comment]
@axoplasm As recently discussed in Slack, I'm concerned that Satsuma is ballooning into a mega-release containing all sorts of overflow from other projects. I'd like to keep that project focused on the theme of integrating the results framework into Tola. 

So we have a new project called Poblano. This is overflow work we'd like to get to soon, that is too big to be a Ghost Pepper hotfix.

As for this ticket, I'm wondering if we need to separate out the "convert to React" work from the design work discussed here... Or if you'd like to keep it all one ticket.

**Comment by jennyhavah** [comment]
As discussed last week, moving anything that still needs design work or is otherwise not "shovel ready" into Fruit Basket. (In this case, design is still in progress and there may be a fairly substantial dependency of first converting the charts to React.)

**Comment by jennyhavah** [comment]
2/11/2020 -- Our MEL intern Mikaela suggested we explore ways of making it more obvious which filter is "active" within the gas tank chart itself... Right now, we do this when you click on any of the "on track" filtering elements:

![image](https://user-images.githubusercontent.com/33670923/74292032-bcfdda00-4ceb-11ea-94b1-27ee50f55a5f.png)

The yellow box tells you one of the filters in that chart is selected, but not _which_ one.
